index.js

```diff
@@ -0,0 +1,2 @@
+require('babel-register');
```

Comment:
The documentation for babel-register states that it be made such, but if you say you're uncomfortable with it, I'll figure out another way around it. My project directory has a dist folder into which babel register pipes my transpiled files. Basically, `require(babel-register)` is the only `require` required to transpile the following `require` statements (in this case `require(./app.js)` into ES6. But then, I'll figure a way around that.